app/routes.py

```diff
@@ -1,2 +1,50 @@
-from flask import Blueprint
+from crypt import methods
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.color = color
+
+    def to_json(self):
+        return {
+            "id": self.id,
+            "name": self.name,
+            "description": self.description,
+            "color": self.color
+        }
+
+planets = [Planet(1, 'Mercury', 'closest to sun', 'gray'),
+Planet(2, 'Mars', 'closest to Earth', 'red'),
+Planet(3, 'Earth', 'our home', 'blue-green')]
+
+
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planet_bp.route("", methods=["GET"])
+def get_planets():
+    planet = []
+    for item in planets:
+        planet.append(item.to_json())
```

Comment:
Nice use of the helper instance method! 

You can also use list comp here like:

`planets_response = [planet.to_json() for planet in planets]`